library/Fission/Internal/Orphanage/OctetStream.hs

```diff
@@ -11,6 +12,17 @@ instance MimeRender OctetStream a => MimeRender OctetStream [a] where
   mimeRender proxy values = "["<> meat <>"]"
     where
       meat :: Lazy.ByteString
-      meat = values
+      meat =
+        values
+          |> fmap (mimeRender proxy)
+          |> Lazy.intercalate ","
+
+instance MimeRender OctetStream a => MimeRender OctetStream (NonEmpty a) where
+  mimeRender proxy values = "["<> meat <>"]"
```

Comment:
what about 
```
instance MimeRender OctetStream a => MimeRender OctetStream (NonEmpty a) where
  mimeRender proxy values = mimeRender proxy <| NonEmpty.toList values
```
to make this a little more dry

Reply:
Oh yeah 👍